modules/GraphvizLayout/src/main/java/org/icculus/chunky/gephigraphviz/GraphvizLayout.java

```diff
@@ -0,0 +1,312 @@
+                    null,
+                    "GraphvizLayout.overlap.name",
+                    NbBundle.getMessage(getClass(), "GraphvizLayout.overlap.name"),
+                    "getOverlap", "setOverlap"));
+
+            properties.add(LayoutProperty.createProperty(
+                    this, Boolean.class,
+                    NbBundle.getMessage(getClass(), "GraphvizLayout.concentrate.desc"),
+                    null,
+                    "GraphvizLayout.concentrate.name",
+                    NbBundle.getMessage(getClass(), "GraphvizLayout.concentrate.name"),
+                    "isConcentrate", "setConcentrate"));
+        } catch (MissingResourceException e) {
+            Exceptions.printStackTrace(e);
+        } catch (NoSuchMethodException e) {
+            Exceptions.printStackTrace(e);
+        }
+        return properties.toArray(new LayoutProperty[0]);
+    }
+
+    @Override
+    public void resetPropertiesValues() {
+    }
+
+    public String getAlgoName() {
+        return algoName;
+    }
+
+    public void setAlgoName(String algoName) {
+        this.algoName = algoName;
+    }
+
+    public String getDotBinary() {
+        return dotBinary;
+    }
+
+    public void setDotBinary(String dotBinary) {
+        this.dotBinary = dotBinary;
+    }
+
+    public String getRankDir() {
+        return rankDir;
+    }
+
+    public void setRankDir(String rankDir) {
+        this.rankDir = rankDir;
+    }
+
+    public boolean isConcentrate() {
+        return concentrate;
+    }
+
+    public void setConcentrate(Boolean concentrate) {
+        this.concentrate = concentrate;
+    }
+
+    public String getOverlap() {
+        return overlap;
+    }
+
+    public void setOverlap(String overlap) {
+        this.overlap = overlap;
+    }
+
+    private void processOutput(final Process dotprocess) {
+        assert dotprocess != null;
+        try (InputStream in = dotprocess.getInputStream()) {
+
+            final BufferedReader outputFromGraphviz = new BufferedReader(new InputStreamReader(in));
+            StringBuilder entireOutput = new StringBuilder();
+            String line;
+            while ((line = outputFromGraphviz.readLine()) != null) {
+                entireOutput.append(line);
+                entireOutput.append("\n");
+//                System.out.println("line");
+            }
+            
+            final String regex = "^\\s*(?<nodeid>\\S+)\\s+\\[.*?[, ]?pos=\"(?<pos>[^\"]+?)\".*?\\]";
+//            System.out.println(entireOutput);
+            final Pattern pat = Pattern.compile(regex, Pattern.MULTILINE | Pattern.CASE_INSENSITIVE | Pattern.DOTALL);
+            Matcher matcher = pat.matcher(entireOutput.toString());
+            while(matcher.find()) {
+                final String nodeid = matcher.group("nodeid");
+                
+                final Node n = graph.getNode(nodeid);
+                if(null == n) {
+                    System.err.println("Cannot find nodeid " + nodeid);
```

Comment:
It would be cleaner to use the logging system for all the `System.out` and `System.err`. You can use `Logger.getLogger("").info(msg);` or `Logger.getLogger("").warn(msg)`